poker-pals/src/App.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <div id="app">
     <router-view></router-view>
+
+    <HelloWorld/>
```

Comment:
Are you using the HelloWorld component for a reason? As opposed to using your own custom component...?